docs/README.md

```diff
@@ -23,9 +23,17 @@ Hugo is available for many platforms. It can be installed using:
 Once installed, you may run the following from the `docs/site` directory
 to access a rendered view of the documentation:
 
-```bash
-hugo server --disableFastRender
-```
+1. Run
 
-Access the site at [http://localhost:1313](http://localhost:1313). Press
+    ```sh
+    make preview-build
```

Comment:
We should have a note saying only to do this step if you want the full versioned docs beyond just the current "edge" version.